src/stream/src/executor/merge.rs

```diff
@@ -160,6 +175,10 @@ impl MergeExecutor {
                     Message::Chunk(_) => {
                         // We may still receive message from this channel.
                         active.push(from.into_future());
+                        {
+                            self.actor_context.lock().info[self.actor_context_position]
```

Comment:
It seems that we can add a flag and only call this method after finished the last barrier. Or it may cost a lot CPU to lock mutex frequently